api/USERS_GUIDE.md

```diff
@@ -281,4 +281,59 @@ current holds for the project:
     "P1.proj_waiting",
     "P1.proj_avail"
 ]
-```
\ No newline at end of file
+```
+
+### Proofreading
+
+The proofreading APIs require retaining and passing in state information. The
+following examples use [jq](https://jqlang.github.io/jq/) to parse responses
+and store them as environment variables.
+
+Get a project's details and parse out the project state:
+```bash
+curl -s -H "Accept: application/json" -H "X-API-KEY: $API_KEY" \
+    -X PUT "https://www.pgdp.org/api/v1/projects/projectID44de3936807f1" \
+    > project.json
+
+PROJECT_STATE=$(cat project.json | jq -r .state)
+```
+
+Check out a page and parse out the page name, state, and page text:
+```bash
+curl -s -H "Accept: application/json" -H "X-API-KEY: $API_KEY" \
+    -X PUT "https://www.pgdp.org/api/v1/projects/projectID44de3936807f1/checkout?state=$PROJECT_STATE" \
+    > proof_page.json
+
+PAGE_NAME=$(cat proof_page.json | jq -r .pagename)
+PAGE_STATE=$(cat proof_page.json | jq -r .pagestate)
+cat proof_page.json | jq '{text}' > page_text.json
+```
+
+Validate the page text doesn't have any weird characters:
+```bash
+curl -s -H "Accept: application/json" -H "X-API-KEY: $API_KEY" \
+    -X PUT "https://www.pgdp.org/api/v1/projects/projectID44de3936807f1/validatetext" \
+    -d @page_text.json
+```
+
+Save the page as in-progress:
+```bash
+curl -s -H "Accept: application/json" -H "X-API-KEY: $API_KEY" \
+    -X PUT "https://www.pgdp.org/api/v1/projects/projectID44de3936807f1/pages/$PAGE_NAME?state=$PROJECT_STATE&pagestate=$PAGE_STATE&pageaction=save" \
+    -d @page_text.json
+```
+
+Get the current page state:
+```bash
+curl -s -H "Accept: application/json" -H "X-API-KEY: $API_KEY" \
+    -X GET "https://www.pgdp.org/api/v1/projects/projectID44de3936807f1/pages/$PAGE_NAME" \
+    > page_info.json
+
+PAGE_STATE=$(cat page_info.json | jq -r .pagestate)
+```
+
+Save the page as done:
+```bash
+curl -s -H "Accept: application/json" -H "X-API-KEY: $API_KEY" \
+    -X PUT "https://www.pgdp.org/api/v1/projects/projectID44de3936807f1/pages/$PAGE_NAME?state=$PROJECT_STATE&pagestate=$PAGE_STATE&pageaction=checkin"
```

Comment:
This appears to be missing the continuation `\` and the

```
   -d @page_text.json
```

Though not strictly necessary, if going by the testing set in #975, also missing the trailing page state query, if the intent is to double-check that the page state is what it's supposed to be:

```
# get the current page state
curl -s -H"Accept: application/json" -H "X-API-KEY: $API_KEY" \
    -X GET "https://www.pgdp.org/api/v1/projects/$PROJECT/pages/$PAGE_NAME" \
    > page_info.json

PAGE_STATE=$(cat page_info.json | jq -r .pagestate)
```

Reply:
Also good catch on the missing `-d @page_text.json` -- I failed at the copy/paste/update from the PR.

I didn't include a follow-up GET because it's not strictly necessary and we'd already given an example of that prior in the guide.